test/__tests__/data/pinterest_tag_input.json

```diff
@@ -1412,6 +1412,79 @@
       "Transformations": []
     }
   },
```

Comment:
For these kind of changes, having unit test for `processUserPayload` is more readable & can easily cover wide range of scenarios

Reply:
Added unit tests